src/Scripts/Game/Core/EL_Utils.c

```diff
@@ -214,6 +226,31 @@ class EL_Utils
 
 		return sortedHierachy;
 	}
+
+	//------------------------------------------------------------------------------------------------
+	static int MaxInt(int a, int b)
```

Comment:
Does int not work with Math::Min / Max?

Reply:
It does but the return type is float, which for function overloads or explicitly intended integer division is meh, so I added these two